pkg/swarm/swarm.go

```diff
@@ -120,8 +120,11 @@ type Chunk interface {
 	WithPinCounter(p uint64) Chunk
 	TagID() uint32
 	WithTagID(t uint32) Chunk
-	Stamp() []byte
-	WithStamp([]byte) Chunk
+	Stamp() Stamp
+	WithStamp(Stamp) Chunk
+	Radius() uint8
```

Comment:
The AP does not specify this interface change. Not sure if this whole AP structure is not generating a ton of overhead to just document everything